web/src/app/util/useExamplePhoneNumber.js

```diff
@@ -0,0 +1,18 @@
+import gql from 'graphql-tag'
+import { useQuery } from 'react-apollo'
+
+const query = gql`
+  query($input: String!) {
+    examplePhoneNumber(input: $input)
+  }
+`
+
+export default function useExamplePhoneNumber(countryCode) {
+  const { data } = useQuery(query, {
+    variables: {
+      input: countryCode,
+    },
+  })
+
+  return data && data.examplePhoneNumber
```

Comment:
It may be useful to return data, loading, and error here for the appropriate checks where necessary. That, or just return `queryResult` instead of destructuring it so whichever component is consuming this can use the return value as it pleases